Mathlib/SetTheory/ZFC/Basic.lean

```diff
@@ -340,11 +340,14 @@ instance (x y : PSet) : Inhabited (insert x y).Type :=
   inferInstanceAs (Inhabited <| Option y.Type)
 
 /-- The n-th von Neumann ordinal -/
+@[deprecated "this will be moved into the ZFC/Ordinal file" (since := "2024-08-18")]
```

Comment:
I don't think we should deprecate these until the replacement exists (hopefully this should be soon anyway?)